src/passkit.cs

```diff
@@ -211,24 +212,39 @@ interface PKPayment {
 	[Protocol, Model]
 	[BaseType (typeof (NSObject))]
 	interface PKPaymentAuthorizationViewControllerDelegate {
+		
+		[Deprecated (PlatformName.iOS, 11,0, message: "Use 'DidAuthorizePayment2' instead.")]
 		[Export ("paymentAuthorizationViewController:didAuthorizePayment:completion:")]
 		[EventArgs ("PKPaymentAuthorization")]
+#if !XAMCORE_4_0
 		[Abstract]
+#endif
 		void DidAuthorizePayment (PKPaymentAuthorizationViewController controller, PKPayment payment, 
 #if XAMCORE_2_0
 			Action<PKPaymentAuthorizationStatus> completion);
 #else
 			PKPaymentAuthorizationHandler completion);
 #endif
 
+		[iOS (11,0)]
+		[Export ("paymentAuthorizationViewController:didAuthorizePayment:handler:")]
+		[EventArgs ("PKPaymentAuthorizationResult")]
+		void DidAuthorizePayment2 (PKPaymentAuthorizationViewController controller, PKPayment payment, Action<PKPaymentAuthorizationResult> completion);
```

Comment:
it's because of the events we added
and XAMCORE might not help unless it's done when our minimum support is iOS11